src/main/java/org/zalando/pazuzu/feature/Tag/Tag.java

```diff
@@ -0,0 +1,60 @@
+package org.zalando.pazuzu.feature.tag;
+
+import javax.persistence.*;
+
+/**
+ * Created by vpavlyshyn on 09/06/16.
+ */
```

Comment:
We have this information in our git repo. Please remove redundant ownership comments.